pkg/cloudprovider/aws/instance.go

```diff
@@ -102,9 +101,6 @@ func (p *InstanceProvider) Create(ctx context.Context,
 	if count := len(createFleetOutput.Instances[0].InstanceIds); count != 1 {
 		return nil, fmt.Errorf("expected 1 instance ids, but got %d due to errors %v", count, createFleetOutput.Errors)
 	}
-	if count := len(createFleetOutput.Errors); count > 0 {
-		zap.S().Warnf("CreateFleet encountered %d errors, but still launched instances, %v", count, createFleetOutput.Errors)
```

Comment:
Thoughts on keeping this log line but just not printing the actual errors? Then we know it's happening, but it's not spammy.

Reply:
I don't think this is necessary especially if we're not printing Errors. We will print an error if the instance doesn't get launched which is what we really care about.   If we print something when an error occurs but we launch capacity, that may be a perfectly valid request w/ no issues but causes a red herring to the operator which is difficult to do anything about without the errors anyways.